src/jrd/dfw.epp

```diff
@@ -5782,3 +5784,32 @@ static string get_string(const dsc* desc)
 
 	return string(str, length);
 }
+
+static bool clear_met_cache(thread_db* tdbb, SSHORT phase, DeferredWork* work, jrd_tra* transaction)
+{
+/**************************************
+ *
+ *	i n v a l i d a t e _ c a c h e d _ p r c
```

Comment:
I'm in no way fan of this routines headers, but as you created it, please make the comment match the routine name.